lego/apps/followers/views.py

```diff
@@ -16,7 +18,10 @@ class FollowerBaseViewSet(
     mixins.DestroyModelMixin,
     viewsets.GenericViewSet,
 ):
-    pass
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        permission_handler = get_permission_handler(queryset.model)
```

Comment:
This can also be archived implicitly by updating the permission handler to do this automagically with permissions.